ddtrace/contrib/botocore/patch.py

```diff
@@ -109,52 +110,75 @@ def inject_trace_data_to_message_attributes(trace_data, entry, endpoint=None):
         if endpoint == "sqs":
             # Use String since changing this to Binary would be a breaking
             # change as other tracers expect this to be a String.
-            entry["MessageAttributes"]["_datadog"] = {"DataType": "String", "StringValue": json.dumps(trace_data)}
+            entry["MessageAttributes"]["_datadog"] = {"DataType": "String", "StringValue": json.dumps(datadog_data)}
         elif endpoint == "sns":
             # Use Binary since SNS subscription filter policies fail silently
             # with JSON strings https://github.com/DataDog/datadog-lambda-js/pull/269
             # AWS will encode our value if it sees "Binary"
-            entry["MessageAttributes"]["_datadog"] = {"DataType": "Binary", "BinaryValue": json.dumps(trace_data)}
+            entry["MessageAttributes"]["_datadog"] = {"DataType": "Binary", "BinaryValue": json.dumps(datadog_data)}
         else:
             log.warning("skipping trace injection, endpoint is not SNS or SQS")
     else:
         # In the event a record has 10 or more msg attributes we cannot add our _datadog msg attribute
         log.warning("skipping trace injection, max number (10) of MessageAttributes exceeded")
 
 
-def inject_trace_to_sqs_or_sns_batch_message(params, span, endpoint=None):
+def get_pathway(pin, params):
+    """
```

Comment:
Can we add typing to this function
```suggestion
     # type: ( ....) -> ......
     """
```